languages/go/exercises/concept/constants/.meta/example.go

```diff
@@ -0,0 +1,49 @@
+package _meta
+
+// FixedInterestRate has a value of 5%
+const FixedInterestRate = 0.05
+
+// GetFixedInterestRate returns the FixedInterestRate constant and gives it a type of float32
+func GetFixedInterestRate() float32 {
+	return FixedInterestRate
+}
+
+// DaysPerYear has a value of 365
+const DaysPerYear int = 365
+
+// GetDaysPerYear returns the DaysPerYear constant
+func GetDaysPerYear() int {
+	return DaysPerYear
+}
+
+// Jan-Dec have values of 1-12
+const (
+	Jan = iota + 1
+	Feb
+	Mar
+	Apr
+	May
+	Jun
+	Jul
+	Aug
+	Sep
+	Oct
+	Nov
+	Dec
+)
+
+// GetMonth returns the value for the given month, ie GetMonth(Nov) = 11
+func GetMonth(month int) int {
+	return month
+}
+
+// AccNo type for a string - this is a stub type included to demonstrate how the untyped string constant can be used where this type is required
+type AccNo string
```

Comment:
I don't see a mention of this type of definition in the `introduction.md`, is that right? If so, the student is unlikely to use this feature and as [per the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#example-implementation-file):

> The implementation should only use language features introduced by the exercise or its prerequisites (and their prerequisites, and so on).

Is this feature introduced in one of the exercises prerequisites?